synfig-core/src/modules/lyr_freetype/lyr_freetype.cpp

```diff
@@ -675,7 +684,18 @@ Layer_Freetype::new_face(const String &newfont)
 	}
 
 #ifdef _WIN32
-	possible_font_directories.push_back("C:\\WINDOWS\\FONTS\\");
+	// All users fonts
+	std::string windir = Glib::getenv("windir");
+	if (windir.empty()) {
+		possible_font_directories.emplace_back("C:\\WINDOWS\\FONTS\\");
+	} else {
+		possible_font_directories.emplace_back(windir + "\\Fonts\\");
+	}
```

Comment:
What about:
```c++
	if (windir.empty())
		windir = "C:\\WINDOWS;
	possible_font_directories.emplace_back(windir + "\\Fonts\\");

Reply:
As for me, it's just faster, but I can change it if it's look more clear

Reply:
It's a matter of taste, actually. But I think it's error prone.